package.json

```diff
@@ -66,6 +66,7 @@
     "ember-cli-sri": "2.1.1",
     "ember-cli-terser": "4.0.2",
     "ember-data": "5.3.9",
+    "ember-fetch": "8.1.2",
```

Comment:
We still need `ember-fetch` as a devDependency but that's only because it's required by `ember-apollo-client`.

Ideally we'd have an isolated test app just for testing apollo and wouldn't need to keep `ember-fetch` project-wide.